src/ATen/native/xpu/mkl/SpectralOps.cpp

```diff
@@ -394,11 +365,10 @@ Tensor& _fft_c2c_mkl_out(
     int64_t normalization,
     bool forward,
     Tensor& out) {
-  auto result = _fft_c2c_mkl(
-      self, dim, static_cast<int64_t>(fft_norm_mode::none), forward);
+  auto result = _fft_c2c_mkl(self, dim, normalization, forward);
   at::native::resize_output(out, result.sizes());
-  return impl::_fft_apply_normalization_out(
-      out, result, normalization, result.sizes(), dim);
+  out.copy_(result);
+  return out;
```

Comment:
@CuiYifeng , I'm considering how to avoid the `copy_` here. Is it feasible to query or calculate the result shape first? so that the `out` tensor can be allocated with a proper shape.

Reply:
This idea looks feasible to me. Let me refine it in another PR.